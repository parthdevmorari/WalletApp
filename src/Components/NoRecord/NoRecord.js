import React from 'react';
import {View, Text} from 'react-native';
import styles from './NoRecordStyles';
import {TextInput} from '@Component';

import Icon from 'react-native-vector-icons/Entypo';
import Icon2 from 'react-native-vector-icons/Ionicons';

export const InputField = (props) => {
  const [isVisible, setIsVisible] = React.useState(false);
  const [icon, setIcon] = React.useState('eye-with-line');

  const {
    title,
    mainContainerStyle,textStyle
  } = props;
 
  return (
    <View style={[styles.inputBoxContainer, mainContainerStyle]}>
      {title ?
        <Text style={[styles.inputLableTaxt, textStyle]}>{title}</Text>:
        <Text style={[styles.inputLableTaxt, textStyle]}>{'No Record Found'}</Text>

      }
      
    </View>
  );
};

export default InputField;
