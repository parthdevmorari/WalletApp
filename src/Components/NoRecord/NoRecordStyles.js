import { StyleSheet, Dimensions, StatusBar } from 'react-native';
import { widthPercentageToDP as wp, heightPercentageToDP as hp } from 'react-native-responsive-screen';
import { color } from '../../Constants/Color';
import { theme } from "../../Constants/Theme";

const width = Dimensions.get('window').width;
const styles = StyleSheet.create({
	inputBoxContainer: {
		flex:1,
		justifyContent:"center",
		alignItems:'center',
	},
	inputLableTaxt: {
		fontSize:wp(5),
		
	},
		

});
export default styles;

