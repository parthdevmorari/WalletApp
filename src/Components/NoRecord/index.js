import component from './NoRecord';
export default component;