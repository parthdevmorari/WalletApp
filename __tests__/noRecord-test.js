import 'react-native';
import React from 'react';
import NoRecord from '../src/Components/NoRecord/NoRecord'
import Enzyme, {shallow, mount} from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

Enzyme.configure({adapter: new Adapter()});

describe('NoRecord Component', () => {
  test('NoRecord Component', () => {
    const wrapper = shallow(<NoRecord />);
    expect(wrapper.exists()).toBe(true);
  });
  it('NoRecord  toMatchSnapshot', () => {
    const wrapper = shallow(<NoRecord />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});