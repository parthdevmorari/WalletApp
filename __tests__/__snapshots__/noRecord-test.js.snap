// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoRecord Component NoRecord  toMatchSnapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 37.5,
        },
        undefined,
      ]
    }
  >
    No Record Found
  </Text>
</View>
`;
